phangsPipeline/casaCubeRoutines.py

```diff
@@ -26,11 +26,103 @@
 # CASA stuff
 from . import casaStuff
 
+# Logging
+#from .pipelineLogger import PipelineLogger
+#logger = PipelineLogger(__name__)
+
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 #endregion
 
+#region Check getchunk putchunk memory issue
+
+def check_getchunk_putchunk_memory_issue(
+    infile, 
+    myia=None,
+    return_myia=False,
+    return_data=False,
+    return_mask=False,
+    return_shape=False,
+    huge_cube_workaround=True,
```

Comment:
I'd vote to just drop the `huge_cube_workaround` kwarg as your changes here negate needing it.